### PR TITLE
310-converter-update-for-product-opco-pages

### LINKED ISSUE #310: [Products Opco] Converter update for product opco pages

### PR DESCRIPTION
**Pull request by rgravitvl**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #310 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://310-converter-update-for-product-opco-pages--danaher-ls-aem--hlxsites.hlx.page/

### CODE REVIEW

File: styles/styles.css
Comment (mhaack) [inline styles/styles.css:1]: Ho is this related to the converter change?

File: tools/actions/convert/test/fixtures/product-opco-home-converted.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+      </div>
+      <hr>
+      <div class="carousel">
+        <div>
+          <div>
+            <h2>The world leader in microscopes and scientific instruments</h2>
+            <p><a href="#">Browse Categories</a></p>
+          </div>
+          <div>
+            <img src="https://author-p93411-e849602.adobeaemcloud.com/content/dam/danaher/brand/see-simplier-well-plates.jpg" alt="Leica">
```
Comment (mhaack) [inline tools/actions/convert/test/fixtures/product-opco-home-converted.html:25]: Update the branch to main, with https://github.com/hlxsites/danaher-ls-aem/pull/540 merged this must be updated to `author-dummy.adobeaemcloud.com`

**Review by mhaack — APPROVED** [review]
LGTM, see comments

**Review by davenichols-DHLS — APPROVED** [review]
Good to go, I see also Markus approved.  Thanks.

File: styles/styles.css
Comment (rgravitvl) [reply styles/styles.css:1]: now its removed

File: tools/actions/convert/test/fixtures/product-opco-home-converted.html (hunk 1 above)
Comment (rgravitvl) [reply tools/actions/convert/test/fixtures/product-opco-home-converted.html]: done
